test/fileImport.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+const _ = require('lodash');
+const config = require('config');
+const should = require('should');
+const homedir = require('homedir');
+
+const client = require('./lib/client.js').client;
+const nav = require('./lib/nav/navigation.js')(client, config);
+const micis = require('./lib/auth/micis.js')(client);
+
+describe('Doing various imports', function() {
+    this.timeout(config.defaultTimeout);
+
+    before('initialize', function(done) {
+        client.clientReady.then(function boot() {
+            if (!micis.loggedOn) {
+                micis.logon();
+            }
+            client.call(done);
+        });
+    });
+
+    describe('Import subjects', function(done) {
```

Comment:
These problems are addressed and fixed in the following PRs:
https://github.com/MRN-Code/asmt/pull/300
https://github.com/MRN-Code/micis/pull/564

Right now Margaret is assigned the Asana [sign off](https://app.asana.com/0/8755179914444/233755861603137) task. Once she is done with that, let's code review, merge, then implement tests here rather then create empty tests.